python/golem_task_api/client.py

```diff
@@ -134,43 +134,38 @@ def __init__(
 class ProviderAppClient:
     DEFAULT_PORT: ClassVar[int] = 50006
 
-    @classmethod
-    def _start_service(
-            cls,
+    def __init__(
+            self,
             service: TaskApiService,
-    ) -> ProviderAppStub:
-        host, port = service.start(
-            f'provider {cls.DEFAULT_PORT}',
-            cls.DEFAULT_PORT,
-        )
-        return ProviderAppStub(
+            port: int = DEFAULT_PORT,
+    ) -> None:
+        self._service = service
+        host, port = service.start(f'requestor {port}', port)
```

Comment:
Why do you start requestor in `ProviderAppClient`?

Reply:
Oops, copy / paste error.. fixed